app/data/census_household.json

```diff
@@ -2182,121 +2248,150 @@
+                                        "id": "ethnic-group-wales-answer",
+                                        "condition": "equals",
+                                        "value": "Asian / Asian British"
+                                    }
+                                ]
                             }
                         },
                         {
                             "goto": {
                                 "id": "black-ethnic-group",
-                                "when": {
-                                    "id": "ethnic-group-wales-answer",
-                                    "condition": "equals",
-                                    "value": "Black / African / Caribbean / Black British"
-                                }
+                                "when": [
+                                    {
+                                        "id": "ethnic-group-wales-answer",
+                                        "condition": "equals",
+                                        "value": "Black / African / Caribbean / Black British"
+                                    }
+                                ]
                             }
                         },
                         {
                             "goto": {
                                 "id": "other-ethnic-group",
-                                "when": {
-                                    "id": "ethnic-group-wales-answer",
-                                    "condition": "equals",
-                                    "value": "Other ethnic group"
-                                }
+                                "when": [
+                                    {
+                                        "id": "ethnic-group-wales-answer",
+                                        "condition": "equals",
+                                        "value": "Other ethnic group"
+                                    }
+                                ]
                             }
                         },
                         {
                             "goto": {
                                 "id": "sexual-identity",
-                                "when": {
-                                    "meta" : "variant_flags.sexual_identity",
-                                    "condition": "equals",
-                                    "value":"true"
-                                }
+                                "when": [
+                                    {
+                                        "id": "over-16-answer",
+                                        "condition": "equals",
+                                        "value": "Yes"
+                                    },
+                                    {
+                                        "meta" : "variant_flags.sexual_identity",
+                                        "condition": "equals",
+                                        "value": true
```

Comment:
then you would need "is true" and "is false". i think making the value optional is going down a bad path. the more caveats that are added with regards to the schema structure the more logic will need to be included in the author and the harder the validation of a schema will become.